website/blog/2025-06-06-the-new-dbt-vscode-extension.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "The new dbt VS Code Extension: The Experience We've All Been Waiting For"
+description: "How the new dbt VS Code extension finally delivers the dev experience we've always wanted."
+slug: vscode-extension-experience
+authors: [bruno_lima]
+
+tags: [analytics craft, data ecosystem]
+hide_table_of_contents: false
+
+date: 2025-06-06
+is_featured: true
+---
+# The new dbt VS Code Extension - The Experience We've All Been Waiting For
+
+Hello, community!
+
+My name is Bruno, and you might have seen me posting dbt content on LinkedIn. If you haven't, let me introduce myself. I started working with dbt more than 3 years ago. At that time, I was very new to the tool, and to understand it a bit better, I started creating some resources to help me with dbt learning. One of them, a dbt cheatsheet, was the starting point for my community journey.
+
+I went from this cheatsheet to creating all different kinds  content, contributing and engaging with the community, until I got the dbt community award for two times, and I am very thankful and proud about that.
+
+Since the acquisition of SDF Labs by dbt Labs, I have been waiting for the day that we would see what the result of the fusion of these two companies would be. Spoiler alert: it’s the dbt Fusion engine and its better than I could have expected.
+
+## The dbt developer experience in the pre-fusion-era
+
+If you've ever started a dbt project, chances are your journey began like mine did: cloning jaffle_shop, opening it in VS Code, and running dbt run for the first time (actually the second time, because I know you forgot to run dbt deps in the first one). It is like our initiation process, our ‘hello-world’.
```

Comment:
```suggestion
If you've ever started a dbt project, chances are your journey began like mine did: cloning jaffle_shop, opening it in VS Code, and running dbt run for the first time (actually the second time, because I know you forgot to run `dbt deps` in the first one). It is like our initiation process, our ‘hello-world’.
```